edge-functions/serverside-analytics/README.md

```diff
@@ -0,0 +1,52 @@
+# Serverside Analytics example
```

Comment:
Adding "example" for titles is usually not required.
```suggestion
# Serverside Analytics
```

Reply:
Done 💯